draft-ietf-tls-tls13.md

```diff
@@ -991,9 +992,10 @@ compatibility, the record layer version identifies as simply TLS 1.0.
 Endpoints supporting other versions negotiate the version to use
 by following the procedure and requirements in {{backward-compatibility}}.
 
-Implementations MUST NOT send zero-length fragments of Handshake or Alert
-types. Zero-length fragments of Application data MAY
-be sent as they are potentially useful as a traffic analysis countermeasure.
+Implementations MUST NOT send zero-length fragments of Handshake or
+Alert types, even if those fragments contain padding. Zero-length
+fragments of Application data MAY be sent as they are potentially
```

Comment:
Looks to be a pre-existing typo. A fix to lowercase it here is probably a good idea.